### PR TITLE
imagesize 1.4.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Upstream license: https://github.com/shibukawa/imagesize_py/blob/master/LICENSE.rst
Setup file: https://github.com/shibukawa/imagesize_py/blob/1.4.1/setup.py

The package imagesize is mentioned inside the packages:
sphinx |

Actions:
1. Remove noarch

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 source:
-  url: https://pypi.io/packages/source/i/imagesize/imagesize-{{ version }}.tar.gz
-  sha256: cd1750d452385ca327479d45b64d9c7729ecf0b3969a58148298c77092261f9d
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 69150444affb9cb0d5cc5a92b3676f0b2fb7cd9ae39e947a5e11a36b4497cd4a
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (katietz) [inline recipe/meta.yaml:15]: I think we lack an 'skip: True   # [py<36]'

Comment (katietz) [reply recipe/meta.yaml:15]: sorry py<34 of course

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,16 +20,17 @@ requirements:
...
 test:
   imports:
     - imagesize
   requires:
     - pip
-    - python <3.10
   commands:
     - pip check
+  downstreams:
+    - sphinx  # [not py==310]
```
Comment (katietz) [inline recipe/meta.yaml]: we shouldn't use downstream feature, please comment

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Sure, I agree. Downstream is a great feature for a non-production building stage. I'll remove it

**Review by katietz — APPROVED** [review]

**Pull request merged**